### ISSUE TITLE
ModuleNotFoundError

### ISSUE THREAD

**Issue by LeoSundholm** [timrae/spotify_to_tidal#99]
After installing and running `spotify_to_tidal` i get the following error:
```
Traceback (most recent call last):
  File "/usr/local/bin/spotify_to_tidal", line 5, in <module>
    from spotify_to_tidal.__main__ import main
ModuleNotFoundError: No module named 'spotify_to_tidal'
```

**Comment by RobinHirst11** [comment]
works for me, did you do ```python3 -m pip install -e .``` 

or preferably tell me the steps to reproduce this error?

**Comment by LeoSundholm** [comment]
yes, I did. my repro was basically following the readme. I ran pip install, renamed the example_config file, registered an app on spotify, and added id, secret, uri and username to the config file. then when I run `spotify_to_tidal` I get the error

**Comment by timrae** [comment]
Does `python3 -m spotify_to_tidal` work? This is usually caused by a problem with the `PATH` variable
